src/dolphin/downloadDolphin.ts

```diff
@@ -27,106 +39,63 @@ export async function assertDolphinInstallation(
     await findDolphinExecutable(type);
     log(`Found existing ${type} Dolphin executable.`);
     log(`Checking if we need to update ${type} Dolphin`);
-    const data = await getLatestReleaseData(type);
-    const latestVersion = data.tag_name;
+    const data = await fetchLatestDolphin(type);
```

Comment:
```suggestion
    const dolphinDownloadInfo = await fetchLatestDolphin(type);
```
Can we use a better variable name than `data` here?